client/repoutils.c

```diff
@@ -39,6 +39,105 @@ TDNFRepoMakeCacheDirs(
     goto cleanup;
 }
 
+uint32_t
+TDNFRepoGetMetalink(
```

Comment:
Is this function used anywhere? I could not find its usage.

Reply:
Yeah, not used. But i was keeping it for future use.. but now removed it.. Lets add when required.